### ISSUE TITLE
feature request, 1 concurrent file, but x concurrent chunks of a file

### ISSUE THREAD

**Issue by Gauss23** [FineUploader/fine-uploader#1692]
feature request
traditional

It would be great to allow only 1 file at a time but if a file is big enough for chunks, send multiple chunks. It seems as if the variable maxConnections is a global setting. Would be nice to have to variables: maxConnections to tell FineUploader to send x files at a time and a variable inside of the chunking block to tell FineUploader how many concurrent chunks of one file can be sent in parallel.
Problem is the session locking on the server side, if too many small files arrive in a too short time. Multiple chunks would be no problem.

Best regards,
Gauss

**Comment by rnicholus** [comment]
This sounds complicated and unnecessary. Chunks will already be sent if the file is big enough for chunks. Not clear what general problem you are trying to solve.

**Comment by rnicholus** [comment]
No response after 3 days: closing.
